checker.pl

```diff
@@ -76,17 +76,224 @@
 communication_course("ENGL 121").
 communication_course("SCIE 300").
 
+/*
+check_core_req(["CPSC 110", "CPSC 121", "MATH 100", "MATH 101", "CHEM 121", "PHYS 101", "BIOL 121", "CPSC 210",
+"CPSC 213", "CPSC 221", "MATH 200", "MATH 221", "STAT 241", "CPSC 310", "CPSC 313", "CPSC 320", "CPSC 304",
+"CPSC 312", "CPSC 340", "CPSC 404", "CPSC 410", "CPSC 416"], MT).
+*/
+check_core_req(Transcript, ModifiedTranscript) :-
+    check_first_year_core_req(Transcript, MT1),
+    check_second_year_core_req(MT1, MT2),
+    check_upper_year_core_req(MT2, ModifiedTranscript).
+
+
+% check_first_year_core_req(["CPSC 110", "CPSC 121", "MATH 100", "MATH 101", "CHEM 121", "PHYS 101", "BIOL 121"], MT).
+check_first_year_core_req(Transcript, ModifiedTranscript) :-
+    check_first_year_cpsc_req(Transcript, MT1),
+    check_first_year_math_req(MT1, MT2),
+    check_phys_science_req(MT2, MT3),
+    check_biol_req(MT3, ModifiedTranscript),
```

Comment:
```suggestion
    check_first_year_math_req(MT1, MT2).
```
Since we are following the new requirements of 2019, `phys_science_req` and `biol_req` are checked as part of `breadth_req_check`